bin/reset-local-dev

```diff
@@ -66,30 +66,33 @@ helm install -f "$CONFIG_FILE" \
   "$GIT_ROOT"
 
 
-echo '
-━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+cat <<EOF
+━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 
- Please run the following command to configure kubectl:
+  Astronomer is launching and should be available shortly. Run the following command
```

Comment:
```suggestion
 Run the following command to allow kubectl to access the Astro cluster:
```

Reply:
I wasn't sure what you are referencing. Astronomer is a company, not a product offering. I assumed (perhaps incorrectly) Astro.

Reply:
The repo is called "Astronomer", and for historical reasons so is the name of the software. Marketing calls it something else that is equally ambiguous.